### ISSUE TITLE
TestYSQLMetrics#testStatementTime fails assertion on total time

### ISSUE THREAD

**Issue by tedyu** [yugabyte/yugabyte-db#6207]
ybd --java-test 'org.yb.pgsql.TestYSQLMetrics#testStatementTime'
```
Details:
Expected total time 1604.744011 to be >= 1649.600000 for query 'INSERT INTO test VALUES(1, '1'), (2, '2'), (3, '3'), (4, '4'), (5, '5')'
Stack trace:
java.lang.AssertionError: Expected total time 1604.744011 to be >= 1649.600000 for query 'INSERT INTO test VALUES(1, '1'), (2, '2'), (3, '3'), (4, '4'), (5, '5')'
```

**Comment by tedyu** [comment]
According to the numbers above, elapsed_local_time would have been 2060
That means 456ms was spent on communication - which implies a regression (since it is greater than 20% as stated in the test).

**Comment by tedyu** [comment]
In pg_wrapper.cc, if pgaudit is not loaded, the test passes.

**Comment by ramsrivatsa** [comment]
Fixed by this commit -> [04daa18](https://github.com/yugabyte/yugabyte-db/commit/04daa18c2d0680983d4a462853a6e01b153a91f9)
